apps/a11y-tests/src/tests/__snapshots__/ComponentExamples.test.tsx.snap

```diff
@@ -698,11 +698,151 @@ Array [
     ],
     "message": Object {
       "richText": "Fix any of the following:
-- Element has insufficient color contrast of 2.21 (foreground color: #a6a6a6, background color: #f4f4f4, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1",
-      "text": "Fix any of the following: Element has insufficient color contrast of 2.21 (foreground color: #a6a6a6, background color: #f4f4f4, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1.",
+- Element has insufficient color contrast of 2.43 (foreground color: #a6a6a6, background color: #ffffff, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1",
+      "text": "Fix any of the following: Element has insufficient color contrast of 2.43 (foreground color: #a6a6a6, background color: #ffffff, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1.",
```

Comment:
@statm This concerns me a bit since it shows `background color` changing, but this PR has no differences in component examples and is not otherwise showing changes to component example snapshots... do you know how this could have changed?

Reply:
The expected results have caused CI to fail again. I suspect `a11y-tests` is using `master` as a reference, not `6.0`.